include/ssl_compat.h

```diff
@@ -89,6 +96,7 @@
 #define X509_get0_notAfter(X) X509_get_notAfter(X)
 #endif
 #endif
+#endif
```

Comment:
Seems like there's a very high likelihood of confusion with all of these nested but not indented preprocessor `#if`s.

Should they be indented?

Reply:
It gets confusing pretty quickly with such mazes.

Reply:
Or maybe add things like `/* ! HAVE_WOLFSSL */` to the `#endif`s for clarity?